core/constants.js

```diff
@@ -237,6 +237,24 @@ Blockly.TOOLBOX_AT_LEFT = 2;
  */
 Blockly.TOOLBOX_AT_RIGHT = 3;
 
+/**
+ * ENUM for positioning the DropDownDiv directly below the target.
+ * @const
+ */
+Blockly.DROPDOWNDIV_BELOW = 1;
+
```

Comment:
Since there's not really a need for these enums to be globally accessible and available to developers. You might consider creating the enum in the dropdown div file, something like: 
```

/**
 * The types for positioning a dropdown.
 * @enum {number}
 */
Blockly.DropDownDiv.PositionType = {
  ABOVE: 1,
  BELOW: 2,
  TOP_OF_PAGE: 3,
};
```